scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.adoc

```diff
@@ -6,22 +6,31 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In resource-constrained environments, such as {sno} deployments, it is advantageous to reserve most of the CPU resources for your own workloads and configure {product-title} to run on a fixed number of CPUs within the host. In these environments, management workloads, including the control plane, need to be configured to use fewer resources than they might by default in normal clusters. You can isolate the {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+In resource-constrained environments, such as {sno} deployments, you should isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs by using workload partitioning.
```

Comment:
I would consider moving the workload partitioning bit up in the sentence since it's the main point you are trying to get across - "In resource-constrained environments, such as {sno} deployments, use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs."

Reply:
good call.